android/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,7 @@
 <manifest
   xmlns:android="http://schemas.android.com/apk/res/android"
   package="com.rt2zz.reactnativecontacts" >
+  <uses-permission android:name="android.permission.READ_PROFILE" />
```

Comment:
are adding these permissions strictly necessary? Does it make using the project easier?